flexmeasures/ui/templates/base.html

```diff
@@ -358,10 +357,15 @@
         });
     }
 
+    var combineLegend = 'true';
     {% if active_page == "assets" %}
         var dataPath = '/api/v3_0/assets/' + {{ asset.id }};
         var dataDevPath = '/api/dev/asset/' + {{ asset.id }};
         var datasetName = 'asset_' + {{ asset.id }};
+        {% set total_sensors = asset.sensors_to_show | map(attribute='sensors') | map('length') | sum %}
```

Comment:
We also allow the "sensor" attribute (value is only an int), and we can still have just list of integers. So I'd wager this is only a complete solution if the API responds with only the one ("sensors") format. 
I checked and I don't believe it does.

Reply:
Hehe, I had also checked and I believe it does. I had also tested asset pages with <7 and >7 sensors.

[This](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures%2Fdata%2Fmodels%2Fgeneric_assets.py#L707) is the relevant line building up the listing, always using `"sensors"` for the key.

Also check the docstring of `sensors_to_show` that mentions the format of the returned list [here](https://github.com/FlexMeasures/flexmeasures/blob/main/flexmeasures%2Fdata%2Fmodels%2Fgeneric_assets.py#L633-L641). Although that does mention the key could be named `"sensor"` instead, in case of a single sensor, that doesn't actually happen in the code.

Reply:
Thanks. Okay then, let's go ahead with this.